app/scripts/advocates.R

```diff
@@ -3,40 +3,37 @@ library(tidyverse)
 library(plotly)
 library(sf)
 library(leaflet)
-library(tigris)
 
-
-shape <- tracts(state='10') %>% select(GEOID, NAMELSAD, NAME)
 lat <- 39.1824#39.5393
 lng <- -75.2
 
 
 # Load Data
 acs_hud_de_geojoined <- read_rds("acs_hud_de_geojoined.rds")
-geo_data <- acs_hud_de_geojoined
-geo_data_nogeometry <- geo_data %>% 
-  st_drop_geometry()
-
-advoc_table <- geo_data_nogeometry %>% 
-  #mutate(reported_labels=ifelse(number_reported==-4,"Less than 10 Households","NA")) %>%
-  #mutate(number_reported =replace(number_reported, number_reported<0, 0) ) %>%
-  rowwise() %>%
-  #mutate(above30 = sum(rent_30E, rent_35E, rent_40E,rent_50E),
-  #       above50=rent_50E) %>%
-  filter(popTotEE>0) %>%
-  mutate(above30 = eligible_renters,
-         above50 = sum(rent_50_10kE,rent_50_20kE,rent_50_35kE,rent_50_50kE,rent_50_75kE)) %>%
-  group_by(GEOID) %>%
-  summarize(tot_hh=sum(tot_hhE),reported_HUD=sum(number_reported),rent_above30=sum(above30),rent_above50=sum(above50)) %>%
-  mutate(prop_above30 = (rent_above30/tot_hh)*100,
+geo_data <- acs_hud_de_geojoined 
+
+advoc_table <- geo_data %>% 
+    rowwise() %>%
+    filter(popTotEE>0) %>%
+    mutate(above30 = eligible_renters,
+           above50 = sum(rent_50_10kE, rent_50_20kE, rent_50_35kE, rent_50_50kE, rent_50_75kE)) %>%
+    transmute(GEOID = GEOID,
+              tract,
+              census_tract_label,
+              tot_hh = sum(tot_hhE),
+              reported_HUD = sum(number_reported),
+              rent_above30 = sum(above30),
+              rent_above50 = sum(above50)) %>%
+    mutate(prop_above30 = (rent_above30/tot_hh)*100,
          prop_above50 = (rent_above50/tot_hh)*100,
          prop_reported_HUD = (reported_HUD/tot_hh)*100) %>%
-  mutate_at(vars(prop_above30),~ round(., 2)) %>%
-  mutate_at(vars(prop_above50),~ round(., 2)) %>%
-  mutate_at(vars(prop_reported_HUD),~ round(., 2)) %>%
-  mutate_at(vars(reported_HUD),as.integer) %>%
-  mutate_at(vars(rent_above30),as.integer) %>%
-  mutate_at(vars(rent_above50),as.integer) %>%
+  mutate_at(vars(prop_above30, 
```

Comment:
Just to note that `mutate_at` is superseded by `across` right now.

Reply:
I'll leave them as-is in this PR, but it would be a good idea to move towards across in the future.